src/utils/constants.js

```diff
@@ -1,11 +1,111 @@
+import ReactCountryFlag from "react-country-flag";
+
 export const languages = [
-  { id: 0, name: "Deutsch", icon: "🇩🇪" },
-  { id: 1, name: "Englisch", icon: "🏴󠁧󠁢󠁥󠁮󠁧󠁿" },
-  { id: 2, name: "Spanisch", icon: "🇪🇸" },
-  { id: 3, name: "Portugisisch", icon: "🇵🇹" },
-  { id: 4, name: "Türkisch", icon: "🇹🇷" },
-  { id: 5, name: "Russisch", icon: "🇷🇺" },
-  { id: 6, name: "Französisch", icon: "🇫🇷" },
+  {
+    id: 0,
+    name: "Deutsch",
+    icon: (
+      <ReactCountryFlag
```

Comment:
- id isn't necessary, we can just use the index of the element in the array
- can we use an array with two props and then map this?
```js
export const languages = [
  { name: "Deutsch", countryCode: "de" },
  ...
].map((language) => ({
  ...language,
  icon: (
      <ReactCountryFlag
        className="flag"
        style={{
          width: null,
          height: null,
        }}
        countryCode={language.countryCode}
        svg
      />
    ),
}))
```

Reply:
done